src/test/a11y/a11y.ts

```diff
@@ -26,13 +25,14 @@ const routesNotTested = [
   '/mock-login',
   '/logout',
   '/robots.txt',
+  '/file-publication',
+  '/list-type',
```

Comment:
the not tested routes are for pages that dont requre a11y testing such as health endpoints, these 2 definitely do need testing for accessibility